spec/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Specification PR template
+When creating a specification project release review, create PRs with the content defined as follows.
+
+Include the following in the PR:
+- [ ] A directory in the form <project-name>/x.y where <project-name> is the specificaiton project name
```

Comment:
I think it is best to point to the release artifact, such as https://download.eclipse.org/microprofile/microprofile-config-2.0-RC1/microprofile-config-spec.html
In this case, there is no need to upload the artifacts somewhere else and less error prone.
Therefore, I suggest the form <project-name>-release-version.

Reply:
The https://download.eclipse.org/microprofile/ directory is too flat in my opinion. The content should be structured based on the current platform version with the specifications under that. So there should be a
https://download.eclipse.org/microprofile/4.0 directory that contains the microprofile-config-2.0-RC1 contents for example.

Reply:
See the https://groups.google.com/u/1/g/microprofile/c/azQZccjYTPE discussion

Reply:
I have simplified the format, review again please

Reply:
Since we are talking about individual spec releases. I think for the time being, the file structure for the spec pdf, html and javadoc can be under these kind of format: https://download.eclipse.org/microprofile/<spec>-<version> e.g. https://download.eclipse.org/microprofile/microprofile-config-2.0-RC3/microprofile-config-spec.pdf